### DISCUSSION TITLE
TaskOrchestrationContext.CallHttpAsync and automatic polling in dotnet-isolated

### DISCUSSION THREAD

**Discussion by danny-ifai** [Azure/azure-functions-durable-extension#2775] [Q&A]
Does the `TaskOrchestrationContext.CallHttpAsync` extension method method in dotnet-isolated Durable Functions support automatic polling of 202 responses? 

Specifically, I'm calling an HttpTrigger on a different Azure Function that starts its own durable orchestrator and uses `CreateCheckStatusResponse` to return a 202 status, location header and requisite url. My understanding is that this should 'just work' with the CallHttpAsync extension method but I seem to just get the 202 response and no polling takes place when I run it locally.

I couldn't find clear documentation on this aspect as the [examples](https://learn.microsoft.com/en-us/azure/azure-functions/durable/durable-functions-http-features?tabs=csharp) all seem to be for the functions 2.x framework.

Not sure if it is not yet fully supported for this scenario or whether it is the way I am running it.

Any insights or examples regarding its support or use would be greatly appreciated.

Many thanks, 
Danny

**Comment by lcharanteja** [comment]
Hello @danny-ifai I am also facing the same issue where the server-side polling on HTTP 202 response isn't happening in my isolated worker model function. Did you find any other guidance related to this?

**Reply by danny-ifai** [reply]
HI @lcharanteja, no sadly not! I have opened #2781 for the same problem in the hopes that it might get fixed! 🙏
